tests/conftest.py

```diff
@@ -1596,8 +1621,28 @@ def verify_new_core_dumps(duthost):
     new_core_dumps = list(new_core_dumps)
 
     if new_core_dumps:
-        pytest.fail("Core dumps found. Expected: %s Found: %s. Test failed. New core dumps: %s" % (len(pre_existing_cores),\
-            len(cur_cores), new_core_dumps))
+        reload = True
+        logger.info("Core dumps found. Expected: %s Found: %s. New core dumps: %s" % (len(pre_existing_cores), len(cur_cores), new_core_dumps))
+
+    # Check if the running config is different from the golden config
+    if duthost.stat(path=CONFIG_DB_BAK).get('stat', {}).get('exists', False):
+        result = duthost.command("sonic-cfggen -d --print-data")
+        running_config = json.loads(result['stdout'])
+
+        result = duthost.command("cat {}".format(CONFIG_DB_BAK))
+        config_db_bak = json.loads(result['stdout'])
+
+        if compare_config(running_config, config_db_bak):
+            reload = True
+    else:
+        logger.info("There is no config_db.json.bak on host %s" % duthost.hostname)
+        return
```

Comment:
If there is no backup config and then return, then new core dump will not fail the test.

Reply:
Yes, you are right, I forget this situation, I will add pytest.fail here.